src/App.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component } from 'react';
+import * as firebase from 'firebase';
 import './App.css';
 
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      users: 10
+    };
+  }
+
+  componentDidMount() {
+    const rootRef = firebase.database().ref();
+    const userRef = rootRef.child('user');
+    userRef.on('value', snap => {
+      this.setState({
+        users: snap.val()
+      });
+    });
+  }
+
   render() {
```

Comment:
something that I do is instead of calling `this.state.whatever` down inside the return i put something like:
`const { users } = this.state` under `render() {`. then down on line 28 you're just calling `users`. just a little cleaner IMO. same thing if you have props. then as the state/props grows etc. you just add it into the const like `const { users, whatever } = this.state`. 

Also 👋  I'm Scott. I was in 1410 and work at Bonobos in NYC.